chrome/capabilities.go

```diff
@@ -67,7 +67,7 @@ type Capabilities struct {
 	// Android Chrome WebDriver path "com.android.chrome"
 	AndroidPackage string `json:"androidPackage,omitempty"`
 	// Use W3C mode, if true.
-	W3C bool `json:"w3c"`
+	W3C bool `json:"w3c,omitempty"`
```

Comment:
Since this is a bool and not a bool*, this omitempty prevents this parameter from being set to false, since the default is apparently true. The "TestChrome/IsDisplayed" test passes without this commit and fails with it. Do we need this omitempty?